src/Document.jsx

```diff
@@ -85,8 +85,13 @@ export default class Document extends PureComponent {
     // If rendering is in progress, let's cancel it
     cancelRunningTask(this.runningTask);
 
-    // If loading is in progress, let's destroy it
+    // If pdfjs loading is in progress, let's destroy it
     if (this.loadingTask) this.loadingTask.destroy();
+
+    // If FileReader loading is in progress, let's abort it
+    if (this.loadFromFileTask) {
+      this.loadFromFileTask.abort();
+    }
```

Comment:
Not a fan of this formatting inconsistency, but honestly, that should have been linter's job.

Reply:
changed it